File: backend/src/BABA.Domain/Models/Atleta.cs
namespace BABA.Domain.Models
{
    public class Atleta
    {
        public int AtletaId { get; set; }
        public string Nome { get; set; }
        public string Apelido { get; set; }
        public int Camisa { get; set; }
        public int Posicao { get; set; }
        public bool Comissao { get; set; }
        public string WhatsApp { get; set; }
        public DateTime DataNascimento { get; set; }
        public DateTime DataCadastro { get; set; }
        public List<Mensalidade> Mensalidades { get; set; }
    }
}

File: frontend/web/BABA-WEB/src/app/models/atleta.ts
import { Mensalidade } from "./mensalidade";

export interface Atleta {
  atletaId: number;
  nome: string;
  apelido: string;
  camisa: number;
  posicao: number;
  comissao: boolean;
  whatsApp: string;
  dataNascimento: Date;
  imageUrl: string;
  dataCadastro: Date;
  mensalidades: Mensalidade[];
}
